database/schema.sql

```diff
@@ -0,0 +1,27 @@
+CREATE TABLE items (
+  id INT GENERATED ALWAYS AS IDENTITY PRIMARY KEY,
+  genre_id INT REFERENCES genres(id),
+  author_id INT REFERENCES authors(id),
+  label_id INT REFERENCES labels(id),
+  publish_date DATE,
+  archived BOOLEAN
+)
```

Comment:
As described in the project requirements, properties and associations in the parent table should be added to the child table. Kindly adjust this. Here is a screenshot of the requirement

![databasePR](https://github.com/hernandanielzamora/Ruby-Capstone/assets/100543892/960d5a35-22e7-47b1-801e-68ed8eadc1d4)